includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -84,18 +115,19 @@ public function get_max_filesize() {
 	 *
 	 * @return mixed
 	 */
-	public function process_image( $metadata, $attachment_id ) {
+	public function generate_image_alt_tags( $metadata, $attachment_id ) {
```

Comment:
@dinhtungdu just to add some context here. This function handles both items because the results of `scan_image` return both the captions and tags. The actual creation for tags and alts tags are done by separate functions called by this function. This is a long-winded way of saying that we save another call to the service by doing this all at once. 

I'd be for a name change to the function to better reflect what it does though :)